extensions/blocks/revue/button-preview.js

```diff
@@ -0,0 +1,184 @@
+// @see https://github.com/WordPress/gutenberg/blob/015555fcdf648b13af57e08cee60bf3f3501ff63/packages/block-library/src/button/edit.js
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	ContrastChecker,
+	InspectorControls,
+	PanelColorSettings,
+	RichText,
+	withColors,
+	__experimentalPanelColorGradientSettings as PanelColorGradientSettings,
+	__experimentalUseGradient,
+} from '@wordpress/block-editor';
+import { PanelBody, RangeControl, withFallbackStyles } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+const { getComputedStyle } = window;
+
+const isGradientAvailable = !! __experimentalUseGradient;
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { textColor, backgroundColor } = ownProps;
+	const backgroundColorValue = backgroundColor && backgroundColor.color;
+	const textColorValue = textColor && textColor.color;
+	//avoid the use of querySelector if textColor color is known and verify if node is available.
+	const textNode =
+		! textColorValue && node ? node.querySelector( '[contenteditable="true"]' ) : null;
+	return {
+		fallbackBackgroundColor:
+			backgroundColorValue || ! node ? undefined : getComputedStyle( node ).backgroundColor,
+		fallbackTextColor:
+			textColorValue || ! textNode ? undefined : getComputedStyle( textNode ).color,
+	};
+} );
+
+const MIN_BORDER_RADIUS_VALUE = 0;
+const MAX_BORDER_RADIUS_VALUE = 50;
+const INITIAL_BORDER_RADIUS_POSITION = 5;
+
+function BorderPanel( { borderRadius = '', setAttributes } ) {
+	const setBorderRadius = useCallback(
+		newBorderRadius => {
+			setAttributes( { borderRadius: newBorderRadius } );
+		},
+		[ setAttributes ]
+	);
+	return (
+		<PanelBody title={ __( 'Border settings', 'jetpack' ) }>
```

Comment:
Would it be clear for the user if the title hear was 'Button border settings' ? - I guess it is obvious when they change it, but at a glance they may think they can apply a border to the whole form.

Reply:
Fixed in 5e74e03a9c9d6bafc5c1e3d3a6c53258f9b3920d